client/plots/disco/dt.disco.js

```diff
@@ -0,0 +1,282 @@
+/*
+
+This is a stand-alone wrapper application
+for Circos-like plots. The 'dt' prefix
+refers to the 'dt[#]' variant classification 
+in pp/src/common.js.
+
+*/
+
+import AppState from './app.state'
+import discoDefaults from './defaults'
+import DiscoReference from './reference'
+import DtDiscoSnv from './snv'
+import DtDiscoCnv from './cnv'
+import DtDiscoLoh from './loh'
+import DtDiscoSv from './sv'
+import Labels from './labels'
+import DtDiscoLegend from './legend'
+import DtDiscoEvents from './events'
+import chord from './chord.js'
+import { select, selectAll } from 'd3-selection'
+import * as controls from './controls'
+import { mclass } from '#shared/common'
+import { sayerror, hideerrors } from '#src/client'
+
+// these are shared across instances
+const mlabel = {}
+for (var key in mclass) {
+	mlabel[mclass[key].label] = mclass[key]
+	mlabel[key] = mclass[key]
+}
+
+const fusionColocationColors = {
```

Comment:
this is defined in multiple scripts

Reply:
I've removed it from this script. It was not used.